core/io/file_access_pack.h

```diff
@@ -108,14 +111,16 @@ class PackedData {
 	void _free_packed_dirs(PackedDir *p_dir);
 
 public:
+	static bool file_require_encryption(const String &p_name);
```

Comment:
```suggestion
	static bool file_requires_encryption(const String &p_name);
```
Grammar nitpick, though it's internal